.github/workflows/sandbox-deleting.yml

```
@@ -3,7 +3,7 @@ name: Trigger Sandbox Deletion
 on:
   pull_request:
     types:
-      - closed
+      - [closed]
```

actionlint (1.7.7)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)